util/guard1/guard1.pact

```diff
@@ -0,0 +1,91 @@
+;; guards.pact
+
+(namespace 'util)
+
+(module guard1 AUTONOMOUS
```

Comment:
I would suggest initially uploading this module to mainnet with an upgradable governance and a big comment prominently warning people not to use this module until it is non-upgradable.

Reply:
Added